ui/app/components/namespace-picker.hbs

```diff
@@ -6,7 +6,12 @@
 <div class="namespace-picker side-padding-4" ...attributes>
   <Hds::Dropdown @enableCollisionDetection={{true}} as |D|>
 
-    <D.ToggleButton @icon="org" @text={{or this.selected.id "-"}} @isFullWidth={{true}} data-test-namespace-toggle />
+    <D.ToggleButton
+      @icon="org"
+      @text={{or this.selectedNamespace.id "-"}}
+      @isFullWidth={{true}}
+      data-test-namespace-toggle
```

Comment:
There is a dynamic generic selector called data-test-toggle-group or data-test-toggle-input, would you mind using that syntax here? I should probably make this consist throughout.